build/ci/release.yml

```diff
@@ -173,6 +173,7 @@ functions:
         secret_key: ${papertrail_secret_key}
         product: ${product}
         version: ${version}
+        submitter: apix-bot[bot]
```

Comment:
note: this could be ${author}, but the description says:

`The name or username of the employee that submitted the release. The formatting of this is loose; as long as it enables quickly determining who to talk to about a particular release, it is sufficient.`

Since we don't have a way to know via PCT, i'm defaulting to the bot and we can rely on oncall rather than pinging the wrong engineer.